heareval/tasks/pipeline.py

```diff
@@ -556,6 +557,119 @@ def run(self):
         self.mark_complete()
 
 
+class ResampleSubcorpuses(WorkTask):
+    """
+    Aggregates resampling of all the splits and sampling rates
+    into a single task as dependencies.
+
+    Parameter:
+        metadata (ExtractMetadata): task which extracts a corpus level metadata
+    Requires:
+        subsample_splits (list(SubsampleSplit)): task subsamples each split
+    """
+
+    sample_rates = luigi.Parameter()
```

Comment:
luigi.ListParameter()

Reply:
I don't think it really matters -- but it should be a list

Reply:
Sure. Thanks

Reply:
@khumairraj ^^